### PR TITLE
Fix duration documentation for Physical Z

### PR DESCRIPTION
**Pull request by dstrain115**
- Adjust duration of physical Z to 20ns.
- Add disclaimer and pointer to gate duration docs
- Fix other random indenting that sphinx is complaining about.

Fixes: #3354

### CODE REVIEW

File: cirq/experiments/cross_entropy_benchmarking.py [hunk 1]
```diff
@@ -93,19 +93,19 @@ class CrossEntropyResult:
     May also include results from speckle purity benchmarking (SPB) performed
     concomitantly.
 
-    Attributes:
+    Args:
```
Comment (viathor) [inline cirq/experiments/cross_entropy_benchmarking.py:96]: This section describes the attributes of the dataclass, so "Args" looks confusing. Is there a tool that complains about this?

**Review by viathor — APPROVED** [review]

File: cirq/experiments/cross_entropy_benchmarking.py (hunk 1 above)
Comment (dstrain115) [reply cirq/experiments/cross_entropy_benchmarking.py]: It looks like the problems were with the multiple indenting, not the section title, so I changed it back to "Atteributes:".  
Thanks for the quick review.

File: cirq/ops/common_channels.py [hunk 2]
```diff
@@ -356,12 +351,9 @@ def depolarize(p: float, n_qubits: int = 1) -> DepolarizingChannel:
...
     This channel evolves a density matrix via
 
-        $$
-        \rho \rightarrow (1 - p) \rho
-            + 1 / (4**n - 1) \sum _i P_i X P_i
-        $$
+    $ \rho \rightarrow (1 - p) \rho + 1 / (4**n - 1) \sum _i P_i X P_i $
 
-        where P_i are the 4**n - 1 Pauli gates (excluding the identity).
+    where P_i are the $4^n - 1$ Pauli gates (excluding the identity).
```
Comment (balopat) [inline cirq/ops/common_channels.py:356]:
```suggestion
    where $P_i$ are the $4^n - 1$ Pauli gates (excluding the identity).
```

File: cirq/experiments/cross_entropy_benchmarking.py [hunk 3]
```diff
@@ -93,19 +93,19 @@ class CrossEntropyResult:
     May also include results from speckle purity benchmarking (SPB) performed
     concomitantly.
 
-    Attributes:
+    Attributes::
```
Comment (balopat) [inline cirq/experiments/cross_entropy_benchmarking.py:96]:
```suggestion
    Attributes:
```

nit: double colon

**Review by balopat — APPROVED** [review]
LGTM - found two nits

File: cirq/experiments/cross_entropy_benchmarking.py (hunk 1 above)
Comment (balopat) [reply cirq/experiments/cross_entropy_benchmarking.py]: We could experiment with the [docparams and docstyle](https://github.com/PyCQA/pylint/blob/master/pylint/extensions/docparams.rst) pylint plugins.

**Pull request merged**